post-workshop/post-workshop-ja.md

```diff
@@ -1,52 +1,52 @@
+*機密保持*
+回答は匿名で記録され、固有のIDを作成するように求められます。ワークショップの事前アンケートに回答した場合は、同じIDを使用してください。 なお、これらの匿名および集計された回答は、学習者の経験を理解し、今後のワークショップを改善し、インストラクターが教育計画に反映するための詳細なフィードバックを提供する目的で、ワークショップのホスト、インストラクター、ヘルパー、カーペントリーズ・コア・チームと共有されます。
 
 
-*Risks and Benefits*
-There are no direct risks or benefits to you from filling out this survey, and no compensation. We hope to use these results to improve workshops for future learners.
+*リスクとメリット*
+このアンケートにご協力いただくことによる直接的なリスクやメリットはなく、報酬もありません。この結果をもとに、今後の学習者のため、ワークショップの改善に役立てたいと考えています。
 
-*Consent*
-You are not required to take this survey to participate in our workshop. You may quit the survey at any time or skip any item other than those required to correctly sort your responses. 
+*承諾*
+このアンケートに回答することは、ワークショップに参加する条件ではありません。アンケートはいつ終了しても構いませんし、回答できる項目以外はスキップしていただいても構いません。
 
-If you have any questions about the study, please contact Erin Becker, Associate Director of The Carpentries at ebecker@carpentries.org or eribecker@ucdavis.edu or Megan Welsh, Assistant Professor of Education at the University of California, Davis at megwelsh@ucdavis.edu.
+この研究についてご質問がある場合は、Carpentriesのアソシエイト・ディレクターであるエリン・ベッカー（ebecker@carpentries.org または eribecker@ucdavis.edu）か、またはカリフォルニア大学デービス校の教育学助教授であるミーガン・ウェルシュ（megwelsh@ucdavis.edu）までご連絡ください。
 
-If you have any questions about your rights or treatment as a research participant in this study or would like to provide input about this research, please contact the University of California at Davis’ Institutional Review Board (IRB) at (916) 703-9151, IRBAdmin@ucdmc.ucdavis.edu, or 2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817.
+本研究の参加者としての権利や待遇について質問がある場合、または本研究について意見を提供したい場合は、カリフォルニア大学デービス校のInstitutional Review Board（IRB）（電話：(916) 703-9151、IRBAdmin@ucdmc.ucdavis.edu、または2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817）までご連絡ください。
 
 -----------
 
-### Do you consent to taking this survey? *
+### このアンケートを受けることに同意しますか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### Are you 18 years of age or above? *
+### 18歳以上の方ですか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### What is the identifier for your workshop? It is in the YYYY-MM-DD-sitename format and is used in the URL for the workshop website or the Etherpad you used.  Please copy and paste the workshop id into the text box below.  *
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename形式で、ワークショップのウェブサイトまたは使用したEtherpadのURLに使用されています。 以下のテキストボックスにワークショップIDをコピーして貼り付けてください。 *
 
-(short text answer)
+(短文回答)
 
-### Please enter a unique identifier as follows: Number of siblings (as numeric) + First two letters of the city you were born in (lowercase) + First three letters of your current street (lowercase).  
+### 固有のIDを以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)
```

Comment:
This seems like a good translation. Do you think we need to be clear that it's alphanumeric (latin alphabet) rather than Japanese characters?

Reply:
I agree. I committed the change: 固有のIDを次のように入力してください：兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（ローマ字の小文字）+ 現在住んでいる通りの最初の3文字 (ローマ字の小文字)

I took "ローマ字", not "アルファベット", but I think both make sense, as long as participants were born in Japan. Please let me know if  "current street" doesn't make sense, because they don't have a street name in address in Japan.

Reply:
Hmm, good point. This may be difficult with Japanese address formats. I think it does not have to an exact translation as long as it generates a unique ID for each participant. Would 市区町村名 be better for Japanese participants? It be my necessary to give separate instructions for participants living in Japan or overseas. 

Note it should be same as the identifier given in the pre-workshop survey that @masamiy is translating (so anything they will remember is okay): https://github.com/swcarpentry-ja/assessment-archives/blob/master/pre-workshop/pre-workshop-ja.md#please-enter-a-unique-identifier-as-follows-number-of-siblings-as-numeric--first-two-letters-of-the-city-you-were-born-in-lowercase--first-three-letters-of-your-current-street-lowercase

Reply:
One option is to give a choice:

固有のIDを次のように入力してください：兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（ローマ字の小文字）+ 現在住んでいる通り**または市町村名**の最初の3文字 (ローマ字の小文字)

I think I should merge @masamiy 's translation of the sentence after she finished.

Reply:
I’m mentioning Masami so she’s notified in case she has any ideas and can make sure the translation is consistent. We can always update your version to match the pre-workshop survey if we’re happy with the translation of everything else. If someone with better understanding of Japanese could review this it would be ideal. I help where I can but I’m a non-native speaker.

Reply:
市町村名 works better, but as Romaji expression always have a vowel in between and 3 characters may not mean much? (Also, I do not know why we should know about their suburbs and .) Can we just ask prefecture / state  (都道府県) they live in?

Reply:
@masamiy I think you've misunderstood the purpose on the question. It's actually to preserve privacy. Giving their street instead of a name gives a "unique identifier" for each participant that shouldn't be traced back to an individual if you don't know where your participants live. I suspect if we ask for ward or prefecture, everyone will answer the same area it we will not be able to match pre- and post- survey answers. Maybe something other than address is better for this in Japan.

Reply:
For reference, Masami has translated this section for the pre-workshop in #6 as:

```
### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)

(短文による回答)
```

Reply:
This is now outdated so marking as resolved.